src/ui.rs

```diff
@@ -775,11 +759,11 @@ impl<'state, 'input> Recorder<'state, 'input> {
                             event: Event::ScrollDown,
                         },
                         MenuItem {
-                            label: Cow::Borrowed("Page up (page-up, ctrl-b)"),
+                            label: Cow::Borrowed("Page up (ctrl-page-up)"),
```

Comment:
suggestion: Given that the label no longer exactly matches the keybinding, let's change it to "Next page" and "Previous page"? (Or "prev", I didn't check to see the current style.) We could also expand on the operation behavior and say "Scroll to next page" if you think it's clearer in some respect.

Reply:
I like this suggestion. The word "previous" is used for other menu items, so I went with that.

- "Previous page"
- "Next page"

I think the intention is clear enough without "scroll to next page", and users can just try it if they're unsure.